packages/d3fc-webgl/src/series/glBar.js

```diff
@@ -32,22 +33,29 @@ import elementConstantAttributeBuilder from '../buffers/elementConstantAttribute
 
 export default () => {
     const program = programBuilder().verticesPerElement(6);
-    let xScale = null;
```

Comment:
@chrisprice I'm unsure whether this change was neccesary. It matches the convention in the errorBar.

Reply:
Yes it is, good spot. We try to provide sane defaults for all values.